Sources/Disposable.swift

```diff
@@ -33,6 +33,7 @@ extension UnsafeAtomicState where State == DisposableState {
 	///
 	/// - returns: `true` if the transition succeeds. `false` otherwise.
 	@inline(__always)
+	@discardableResult
```

Comment:
I don't think this result should be discardable.  It's important that we check the result most of the time.

Reply:
`tryDispose()` is a method for transitioning the state of `Disposable` and thinks that the result is a supplementary thing.
How is it?

Reply:
The returned Boolean indicates whether the caller is the one uniquely transitioned the state, which is critical to most use cases.